tests/unit/test_models/test_base_model.py

```diff
@@ -187,15 +184,15 @@ def test_get_parameter_info(self):
         }
 
         parameter_info = model.get_parameter_info()
-        self.assertEqual(parameter_info["a"][1], "InputParameter")
-        self.assertEqual(parameter_info["b"][1], "InputParameter in ['test']")
-        self.assertIn("c", parameter_info)
-        self.assertIn("d", parameter_info)
-        self.assertIn("e", parameter_info)
-        self.assertIn("f", parameter_info)
-        self.assertEqual(parameter_info["g"][1], "Parameter")
-        self.assertIn("h", parameter_info)
-        self.assertIn("i", parameter_info)
+        assert parameter_info["a"][1] == "InputParameter"
+        assert parameter_info["b"][1] == "InputParameter in ['test']"
+        assert "c" in parameter_info
+        assert "d" in parameter_info
+        assert "e" in parameter_info
+        assert "f" in parameter_info
+        assert parameter_info["g"][1] == "Parameter"
+        assert "h" in parameter_info
+        assert "i" in parameter_info
```

Comment:
Wherever you find repetition like this, it's probably better to parameterize the function and gain benefits, both in terms of readability and slightly faster execution by parallelization.